### DISCUSSION TITLE
help (polymod error)

### DISCUSSION THREAD

**Discussion by Benby3** [KadeArchive/Kade-Engine#2897] [Q&A] [answered]
when i build it this happen `C:/HaxeToolkit/haxe/lib/polymod/git/polymod/fs/PolymodFileSystem.hx:19: characters 16-18 : Expected )`

how my haxelib list looks like
![image](https://user-images.githubusercontent.com/96499074/161392017-daf9c495-8b4b-4d2b-8786-6d8e94863704.png)

**Comment by Benby3** [comment, marked as answer]
nvm i figured out

**Reply by hifishlol** [reply]
to be honest with you, i would just get rid of polymod as a whole

**Reply by Benby3** [reply]
it's more easy just delete and re-download everything
